RELEASE-NOTES.rst

```diff
@@ -0,0 +1,37 @@
+=============================
+ Invenio-OAIHarvester v0.1.0
+=============================
+
+Invenio-OAIHarvester v0.1.0 was released on August 14, 2015.
+
+About
+-----
+
+The OAI Harvest module handles metadata gathering between OAI-PMH v.2.0
```

Comment:
`Invenio module for OAI-PMH metadata harvesting between repositories.`

see `setup.py` fix from @tiborsimko + we should fix also the `README.rst`